packages/design-preview/src/components/sidebar.tsx

```diff
@@ -93,23 +90,7 @@ const Sidebar: React.FC< SidebarProps > = ( {
 						) }
 						{ ( decodedDescription || decodedShortDescription ) && (
 							<div className="design-preview__sidebar-description">
-								<p>
-									{ isShowDescriptionToggle ? (
-										<>
-											{ isShowFullDescription ? decodedDescription : decodedShortDescription }
-											<Button
-												borderless
-												onClick={ () => setIsShowFullDescription( ! isShowFullDescription ) }
-											>
-												{ isShowFullDescription
-													? translate( 'Read less' )
-													: translate( 'Read more' ) }
-											</Button>
-										</>
-									) : (
-										decodedDescription ?? decodedShortDescription
-									) }
-								</p>
+								<p>{ screens.length !== 1 ? decodedDescription : decodedShortDescription }</p>
```

Comment:
> Themes support colors and fonts but not style variations

The condition doesn't match this scenario. If a theme supports all of them, we also show the full description 😆

Reply:
Sorry could you show me actual counterexample of this? Which theme shows full description where it shouldn't?